README.md

```diff
@@ -1,116 +1,108 @@
 # IPFS Blog
 
-[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](https://protocol.ai)
 [![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
 [![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-> Source for the [IPFS Blog](http://ipfs.io/blog)
+> Source for the [IPFS Blog](https://blog.ipfs.io)
 
-![](https://www.evernote.com/l/AMaEbN3YfmVC-JDtlxRdFnMMbfvQjQlmU9MB/image.png)
+![ipfs-blog @ 2018-11-30](https://user-images.githubusercontent.com/157609/49290285-e21b7980-f4a6-11e8-8829-fe8977b5be34.png)
```

Comment:
While you are at it, could you pin the image on IPFS?

Reply:
done ✨